docs/data-ai/reference/data/_index.md

```diff
@@ -25,35 +25,6 @@ toc_hide: true
 The data management service captures data from one or more {{< glossary_tooltip term_id="resource" text="resources" >}} locally, and syncs it to cloud storage when a connection to the cloud is available.
 You can configure which data you want to capture, as well as the capture rate and the sync frequency.
 
-{{< tabs >}}
```

Comment:
I was unsure about moving this over to how sync works. It does describe how sync works in addition to capture

Reply:
this seems fine to me, you can link to it from here for further reading.